libretroshare/src/tcponudp/tou.cc

```diff
@@ -265,7 +265,7 @@ int 	tou_connect(int sockfd, const struct sockaddr *serv_addr,
 		return -1;
 	}
 #else
-	UdpPeerReceiver *upr = (UdpPeerReceiver *) (tous->udpsr);
+	UdpPeerReceiver *upr = reinterpret_cast<UdpPeerReceiver *>(tous->udpsr);
```

Comment:
reinterpret_cast is extreme. If you can, use static_cast<>.

Reply:
Done